klimenko/Lesson3_4_Klimenko/src/main/java/controller/AppartmentsController.java

```diff
@@ -0,0 +1,63 @@
+package controller;
+
+import dao.impldao.ImplApartmentsDAO;
+import entity.Apartments;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(name = "Appartments", urlPatterns = {"/appartments/*"})
+public class AppartmentsController extends HttpServlet {
+
+    ImplApartmentsDAO apartmentDAO;
```

Comment:
If you use interface here, you can add interface field, instead of implementation